virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -4,15 +4,18 @@ include::modules/virt-document-attributes.adoc[]
 :context: preparing-cluster-for-virt
 toc::[]
 
```

Comment:
This is now a Note followed by two bullet points. Should we have an introductory sentence?